torch/types.py

```diff
@@ -34,13 +34,25 @@
 class Storage(object):
     _cdata: int
 
+    def __deepcopy__(self, memo) -> 'Storage':
+        ...
+
+    def _new_shared(self, int) -> 'Storage':
+        ...
+
     def _write_file(self, f: Any, is_real_file: _bool, save_size: _bool) -> None:
         ...
 
-    def size(self) -> int:
+    def element_size(self) -> int:
         ...
 
-    def _new_shared(self, int) -> 'Storage':
```

Comment:
Why are you deleting _new_shared() here?

Reply:
I was reordering them based on alphabetical order :-)